tests/run/generators.pyx

```diff
@@ -561,3 +561,19 @@ def test_generator_kwds3(**kwargs):
     a
     """
     yield from kwargs.keys()
+
+def test_generator_scope():
```

Comment:
Why have this twice? It seems enough to include it in the Python compatibility test file.

Reply:
Only because the generators.pyx and generators_py.py already look to be duplicates of each other so I was copying what was already there. I agree there's no real benefit.